.github/workflows/prombench-gmp-collector.yml

```diff
@@ -126,9 +128,18 @@ jobs:
             -H "X-GitHub-Api-Version: 2022-11-28" \
             https://api.github.com/repos/${GITHUB_ORG}/${GITHUB_REPO}/issues/${PR_NUMBER}/comments \
             --data '{ "body": "'"${COMMENT_MSG}"'" }'
-
+          
+      - name: 'Adding the prombench label to the PR'
+        if: env.CANCEL_BENCHMARK == 'false'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          curl --silent -L -X POST -H "Accept: application/vnd.github+json" -H "Authorization: Bearer $GITHUB_TOKEN" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            https://api.github.com/repos/${GITHUB_ORG}/${GITHUB_REPO}/issues/${PR_NUMBER}/labels \
+            -d '{"labels":["prombench"]}' || true
```

Comment:
what is `|| true` doing?

Reply:
It's just to ensure that even if the command fails for some reason, the workflow does not stop. If the add label command fails, the exit code is non zero which will stop the remaining steps in the workflow. The `|| true ` ensures that the exit code is always `0`.

Reply:
Wouldn't you need it for the comment in line 122.

Reply:
Let's add failure resistance when we migrate from curl to gh